src/browser_launchers/browser_launcher.js

```diff
@@ -27,25 +72,29 @@ export default class BrowserLauncher {
         this.useChrome = useChrome;
         this.launchOptions = launchOptions;
 
-        this.Plugin = null; // tight to specific library implementation;
+        this.Plugin = null; // to be provided by child classes;
     }
 
     /**
-     * @return {BrowserPlugin}
+     * @returns {BrowserPlugin}
+     * @private
```

Comment:
If it's private, the function name should be prefixed with `_`. If it's public, but we don't want to display it in Docs, it should have `@ignore`.